### PR TITLE
Align vscode-ws-jsonrpc README and server example

### PR DESCRIPTION
**Pull request by kaisalmen**
This PR was triggered by #445 and corrects the example in the README.

I have polished the server example along and discovered something that we should discuss in another issue (#447).

### CODE REVIEW

File: packages/examples/server/src/server.ts [hunk 1]
```diff
@@ -3,17 +3,12 @@
...
 import express from 'express';
-import * as rpc from 'vscode-ws-jsonrpc';
-import { launch } from './json-server-launcher.js';
-
-// solve: __dirname is not defined in ES module scope
-import path from 'path';
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = path.dirname(__filename);
+import { IWebSocket } from 'vscode-ws-jsonrpc';
+import { getLocalDirectory, launch } from './json-server-launcher.js';
```
Comment (CGNonofr) [inline packages/examples/server/src/server.ts:11]: I'm not sure `json-server-launcher` is the right place to put the `getLocalDirectory` method

Comment (kaisalmen) [reply packages/examples/server/src/server.ts:11]: It is used there as well. Any good idea where to put it? New file and both reference that one?

Comment (CGNonofr) [reply packages/examples/server/src/server.ts:11]: That's probably what I would have done, sbut I don't know how to name it :grin:

Comment (kaisalmen) [reply packages/examples/server/src/server.ts:11]: Ok, I go for " iDontKnowHowToNameIt.ts". 😆 No, seriously, I do that and come up with a hopefully fitting name.

**Comment by kaisalmen** [comment]
Done. Btw, are you generally ok with squash and rebase of PRs? I should not be a "must", but it helps to keep the git history graph a bit cleaner. It is not pressing here because of small project size. I just want to know what you think?

**Review by CGNonofr — APPROVED** [review]

**Pull request merged**